docs/bok/Activities/Level-2/Creating-An-OSPO.md

```diff
@@ -4,21 +4,169 @@ tags:
   - OSPO (Role)
   - Placeholder
   - Level 2 (OSMM)
-draft: true
 sidebar_position: 7
 ---
 
+- Lots of the level 2 activities are probably being performed piecemeal by the organisation.
+- Creating an OSPO allows you to centralise and structure all the level 2 activities and allow you to start working on Level 3.
 
+An OSPO is designed to be the center of competency for an organization's open source operations and structure as defined by the [TODO group.](https://github.com/todogroup/ospodefinition.org). Creating an OSPO is _organisational change_. Therefore, in this article, we are going to use the formalism of [Kotter's 8-Step Process for Leading Change](https://en.wikipedia.org/wiki/Change_management#Change_models) to structure this article, however, you can approach this any way you want.
```

Comment:
Heads up on this: the open source community is working on a new version of the definition that reflects current OSPO trends and best practices. For instance, it now also looks at the *OSPO* from the *human* side of the term (The people behind the OSPO) and deep dives into the *what*, *why*, *who*, and *how*. I'd suggest updating the definition The PR related is: https://github.com/todogroup/ospodefinition.org/pull/9

On a side note, if contributors from this group would like to help with the 2023 release, please leave your suggestions/comments in the PR 😊

Reply:
linked, thanks!